src/transformers/utils/versions.py

```diff
@@ -0,0 +1,86 @@
+"""
+Utilities for working with package versions
+"""
+
+import operator
+import re
+import sys
+
+from packaging import version
+
+import pkg_resources
+
+
+ops = {
+    "<": operator.lt,
+    "<=": operator.le,
+    "==": operator.eq,
+    "!=": operator.ne,
+    ">=": operator.ge,
+    ">": operator.gt,
+}
+
+
+def require_version(requirement, hint=None):
+    """
+    Perform a runtime check of the dependency versions, using the exact same syntax used by pip.
+
+    The installed module version comes from the `site-packages` dir via `pkg_resources`.
+
+    Args:
+    - requirement: pip style definition, e.g.,  "tokenizers==0.9.4", "tqdm>=4.27", "numpy"
+    - hint: what suggestion to print in case of requirements not being met
```

Comment:
Super nit, but since you're writing a docstring, can we have it match our usual formatting? Also we try to add type-annotations on new methods now.

Reply:
Done. Please kindly check if I've missed something. Thank you, @sgugger!